packages/@aws-cdk/aws-redshift/package.json

```diff
@@ -82,20 +82,52 @@
     "@aws-cdk/assert": "0.0.0",
     "cdk-build-tools": "0.0.0",
     "cfn2ts": "0.0.0",
+    "jest": "^24.9.0",
     "pkglint": "0.0.0"
   },
   "dependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-secretsmanager": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.0.2"
   },
   "homepage": "https://github.com/aws/aws-cdk",
   "peerDependencies": {
+    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-secretsmanager": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.0.2"
   },
   "engines": {
     "node": ">= 10.12.0"
   },
+  "awslint": {
+    "exclude": [
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupFamily.REDSHIFT_1_0",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupFamily.DUMMY",
+      "docs-public-apis:@aws-cdk/aws-redshift.ClusterType.MULTI_NODE",
+      "docs-public-apis:@aws-cdk/aws-redshift.ClusterType.SINGLE_NODE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC1_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC1_LARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC2_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DC2_LARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DS2_8XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.DS2_XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.NodeType.RA3_16XLARGE",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupParameters.parameterName",
+      "docs-public-apis:@aws-cdk/aws-redshift.ParameterGroupParameters.parameterValue",
```

Comment:
Can we add these docs? I now they can be a pain, but it would be great to have them either way (for example, the enum ones can talk about which values they correspond to in CFN).